rotkehlchen/api/v1/schemas.py

```diff
@@ -546,6 +546,10 @@ def validate_ledger_action_schema(
                 field_name='identifier',
             )
 
+        lastest_valid_date = ts_now()
+        if lastest_valid_date is not None and data['timestamp'] > lastest_valid_date:
+            raise ValidationError(f'Given date has to be earlier than {lastest_valid_date}')
```

Comment:
simplify. ts_now() can't be None and giving a timestamp number does not really help the user understand more than just saying "in the future"

```suggestion
        if data['timestamp'] > ts_now():
            raise ValidationError('Given date can't be in the future')
```